src/borg/archiver.py

```diff
@@ -2168,6 +2168,13 @@ def main():  # pragma: no cover
     except RemoteRepository.RPCError as e:
         msg = '%s\n%s' % (str(e), sysinfo())
         exit_code = EXIT_ERROR
+    except BrokenPipeError:
+        if args.func.__name__ in ['do_info', 'do_list', 'do_diff']:
```

Comment:
ah, interesting approach. so the idea is that for all known "read only" commands, we ignore the BrokenPipeError?

did you try to find out which pipe was broken?